ansible/aws/README.md

```diff
@@ -1,5 +1,14 @@
 # Description
 
+## Prerequisites
+1. Ansible 2.9
+2. Collection amazon.aws (can be installed by command `ansible-galaxy collection install amazon.aws`)
+3. Created key pair and public key stored in `AWS.pub` in same folder as the `aws-provisioning.yaml` playbook
```

Comment:
```suggestion
1. Created key pair and public key stored in `AWS.pub` in same folder as the `aws-provisioning.yaml` playbook
```

It may look weird but that's the recommended approach in markdown